### PR TITLE
feat:Add website v2

### PR DESCRIPTION
**Pull request by vitalboyzf**
closes: #ISSUE_Number

---

## Change logs

> Describe your change clearly, including what problem is being solved or what document is being added or updated.

## Contributor's checklist

Here are some reminders before you submit your pull request:

* Make sure that your Pull Request has a clear title and commit message. You can take the [Git commit template](https://github.com/cloudberrydb/cloudberrydb/blob/main/.gitmessage) as a reference.
* Sign the Contributor License Agreement as prompted for your first-time contribution (*One-time setup*).
* Learn the [code contribution](https://cloudberrydb.org/contribute/code) and [doc contribution](https://cloudberrydb.org/contribute/doc) guides for better collaboration.
* Make sure that your changes deployment preview is successful.
* List your communications in the [GitHub Issues](https://github.com/cloudberrydb/cloudberrydb-site/issues) or [Discussions](https://github.com/orgs/cloudberrydb/discussions) (if has or needed).
* Feel free to ask for the @cloudberrydb/doc team or other people to help review and approve.

### CODE REVIEW

File: src/theme/BlogLayout/index.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from "react";
```
Comment (fengyuchaochao) [inline src/theme/BlogLayout/index.tsx:1]: import导入规范：
如果一个文件import模块比较多时，后期维护过程中，模块查找，引入等就会有一些混乱，因此，建议对import的所有模块进行分类整理，一般可以遵循以下原则：
1. 首先导入node_modules下面的第三方依赖模块，例如：react,clsx等
2. 其次导入当前项目的公共依赖，通过是利用@别名，例如：@/theme等
3. 最后导入当前模块所在目录下，利用相对路径导入的模块。
4. 同时，各个类别下的模块，如果依然比较多，可以采用字母顺序去进一步排序，同时模块类别较多时，也可以用空格进行分类。
5. 类型模块也可以单独维护在一个代码块中。

针对这里的代码以及优化建议，优化后的建议代码如下：
import React, { useEffect, useState } from "react";
import clsx from "clsx";
import { useLocation } from "@docusaurus/router";

import Layout from "@theme/Layout";
import BreadCrumbs from "@site/src/components/common/Breadcrumbs";

import BlogSidebar from "../BlogSidebar";
import styles from "./styles.module.scss";

import type { Props } from "@theme/BlogLayout";

File: src/components/bootcamp/MainContent/index.tsx [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+      </div>
+
+      <div
+        className={styles.cardLine}
+        style={{
+          marginTop: 25,
+          marginBottom: 32,
+        }}
+      >
+        <Card
```
Comment (fengyuchaochao) [inline src/components/bootcamp/MainContent/index.tsx:48]: 优化建议：尽可能复用公共代码逻辑，

这里，建议把数据单独维护到一个单独的state里，然后通过map循环等方式去渲染最终结构，减少重复代码，代码也会更加简洁

File: src/components/common/Breadcrumbs/index.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  }
+  return (
+    <div
+      className={clsx(styles.breadcrumbs, props.className)}
+      style={props.style}
+    >
+      <LinkWithBaseUrl to={"/"} className={styles.backLogoA}>
+        <BackHome className={styles.backLogoSvg} />
+      </LinkWithBaseUrl>
+      <div className={styles.path}>
```
Comment (fengyuchaochao) [inline src/components/common/Breadcrumbs/index.tsx:45]: 优化建议：嵌套JSX结构比较复杂的时候，尤其是耦合了较多JS逻辑的时候，建议把相关结构单独抽成一个子方法来维护，确保JSX结构整体较为清晰。例如：此处的map循环相关代码内部还有很多其他逻辑，建议把其单独抽成一个子render函数。

**Review by tuhaihe — APPROVED** [review]
Approved.
@vitalboyzf thanks for your great work!

**Pull request merged**